protocol/messenger_handler.go

```diff
@@ -938,6 +938,11 @@ func (m *Messenger) handleAcceptContactRequestMessage(state *ReceivedMessageStat
 		}
 		state.Response.AddMessage(updateMessage)
 
+		chat.UnviewedMessagesCount++
```

Comment:
Isn't  better to increase counter after `UpdateFromMessage` because it can exit with error here?